nixos/tests/elk.nix

```diff
@@ -203,11 +241,13 @@ in pkgs.lib.mapAttrs mkElkTest {
       logstash      = pkgs.logstash7;
       kibana        = pkgs.kibana7;
       journalbeat   = pkgs.journalbeat7;
+      metricbeat    = pkgs.metricbeat7;
     }
     else {
       elasticsearch = pkgs.elasticsearch7-oss;
       logstash      = pkgs.logstash7-oss;
       kibana        = pkgs.kibana7-oss;
       journalbeat   = pkgs.journalbeat7;
-    };
+      metricbeat    = pkgs.metricbeat7;
+  };
```

Comment:
The indention of this line seems to be a bit off...

Reply:
Fixed. Thanks.